File: C++ programming/PROBLEMS ONC NUMBER/Program3.cpp
#include<iostream>
using namespace std;

void DisplayFactor(int iNo)
{
    int iCnt  = 0;
    for(iCnt = 1; iCnt < iNo; iCnt++)
    {
        if((iNo % iCnt ) == 0)
        {
            cout<<iCnt<<endl;
        }
    }
}   
int  main()
{
int iValue = 0;
cout<<"Enter First Number : \n";
cin>>iValue;

DisplayFactor(iValue);
    return 0;
}

File: JAVA Programming/PROBLEMS ON NUMBER/Program3.java
import java.util.*;
class Program3
{
    public static void DisplayFactor(int iNo)
    {
        for(int i = 1; i < iNo; i++)
        {
            if(iNo % i == 0)
            {
                System.out.println(i);
            }
        }
    }
    public static void main(String Args[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter First Value:");
        int iValue = sobj.nextInt();
        DisplayFactor(iValue);
    }
}